src/friendly-iframe-embed.js

```diff
@@ -298,10 +300,20 @@ function mergeHtml(spec) {
     });
   }
 
+  const cdnBase = getMode().localDev ? 'http://localhost:8000/dist' : urls.cdn;
+  const cspScriptSrc = [
+    `${cdnBase}/esm/`,
+    `${cdnBase}/lts/`,
+    `${cdnBase}/mp/`,
+    `${cdnBase}/rtv/`,
+    `${cdnBase}/sp/`,
+    `${cdnBase}/sw/`,
+  ].join(' ');
+
   // Load CSP
   result.push(
     '<meta http-equiv=Content-Security-Policy ' +
-      "content=\"script-src 'none';object-src 'none';child-src 'none'\">"
+      `content="script-src ${cspScriptSrc};object-src 'none';child-src 'none'">`
```

Comment:
Changes to FIE LGTM. I will need to copy this change over to no signing, will send you a follow up PR.